ui/eureka/eureka.ts

```diff
@@ -35,6 +36,15 @@ import './eureka.css';
 // TODO: get all of the elements required up front in the constructor
 // TODO: split NMInfo from some new InitializedNMInfo, which includes required element/timeElement
 
+// TODO: Pull the area names from PlaceName (not currently cached locally, will require expanding the `ZoneInfo` type or adding a new resource file)
```

Comment:
as the linter says, this line is very long, and it should be easy to split it up since its a comment.